frontend/src/Student/pages/Submit.js

```diff
@@ -1,18 +1,26 @@
 import React from 'react'
-import { Link } from 'react-router-dom'
-import { Container, Row, Col, Button } from 'reactstrap'
+import { Route, Switch } from 'react-router-dom'
+import { Container } from 'reactstrap'
+import { connect } from 'react-redux'
 
-import SubmissionForm from '../containers/SubmissionForm'
+import SubmissionFormChooser from '../components/SubmissionFormChooser'
+import PhotoSubmissionForm from '../containers/PhotoSubmissionForm'
+import VideoSubmissionForm from '../containers/VideoSubmissionForm'
+import OtherMediaSubmissionForm from '../containers/OtherMediaSubmissionForm'
 
-const Submit = () => (
+const Submit = (props) => (
   <Container>
-    <Row>
-      <Col>
-        <h1>New Submission</h1>
-        <SubmissionForm />
-      </Col>
-    </Row>
+    <Switch>
+      <Route exact path='/submit' component={SubmissionFormChooser} />
+      <Route exact path='/submit/photo' render={() => <PhotoSubmissionForm forShow={props.forShow} />} />
+      <Route exact path='/submit/video' render={() => <VideoSubmissionForm forShow={props.forShow} />} />
+      <Route exact path='/submit/other' render={() => <OtherMediaSubmissionForm forShow={props.forShow} />} />
+    </Switch>
   </Container>
 )
 
-export default Submit
+const mapStateToProps = (state) => ({
+  forShow: { id: 1, name: 'Honors Show 2017' } // TODO: Get from Redux
```

Comment:
Yes, this is intentional – not really a big issue. When @chtinahow has the Student dashboard complete, when you click "Make Submission" on a particular show, we'll either pass that show's id and name as props or access them through Redux. Since she'll have already queried for the show to display it on the dashboard, we'll just need to pass that data along.

If you want to make an Issue for integrating our two features together, feel free.